### PR TITLE
Add simple script to backport Beam fixes to DataflowJavaSDK

### PR DESCRIPTION
**Pull request by kennknowles**
This is a very simple, very naive, script to automate the short sequence of git commands needed to rebase a feature branch onto the Dataflow Java SDK.

After running the script, the user will need to submit a pull request as usual, to ensure that it is acceptable to Travis, GoogleBot, and a cursory human review.

### CODE REVIEW

File: backport.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
```
Comment (davorbonaci) [inline backport.sh]: Please add comments.

File: backport.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+BEAM_PR=$1
```
Comment (davorbonaci) [inline backport.sh]: Perhaps we don't need this. Could merge commit be enough?

File: backport.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+MERGE_COMMIT=$2
+
+LOCAL_BRANCH="backport-${BEAM_PR}"
+
+git fetch --all
+git rebase --onto dataflow/master "$MERGE_COMMIT"^1 "$MERGE_COMMIT"^2
```
Comment (davorbonaci) [inline backport.sh]: not state-agnostic. dependent on having these remotes.

File: backport.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+git checkout -b "$LOCAL_BRANCH"
```
Comment (davorbonaci) [inline backport.sh]: can the script leave the state back to its original place?

File: backport.sh (hunk 1 above)
Comment (davorbonaci) [inline backport.sh]: move to travis/ directory.

**Comment by dhalperi** [comment]
@kennknowles status or close?

**Comment by kennknowles** [comment]
Yea, I don't think we need to bother.

**Pull request closed without merging**